Public/Src/Engine/Dll/InputTracker.cs

```diff
@@ -530,7 +530,9 @@ private void RegisterFile(string path, ContentHash hash)
         /// </summary>
         public void WriteToFile(
             BinaryWriter writer,
+            PathTable pathTable,
             IReadOnlyDictionary<string, string> buildParametersImpactingBuild,
+            IReadOnlyDictionary<string, IMount> mountsImpactingBuild,

```

Comment:
Nit:perhaps annotate with [CanBeNull] so we get IDE help when calling this function.